samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/GradingService.java

```diff
@@ -3391,6 +3428,26 @@ private boolean isCalcQuestion(List tempItemGradinglist, Map publishedItemHash)
 	  }
 	  return false;
   }
+  
+  /**
+   * IMAGE_MAP QUESTION
+   * Simple to check to see if this is a calculated question. It's used in storeGrades() to see if the sort is necessary.
+   */
+  private boolean isImageMapQuestion(List tempItemGradinglist, Map publishedItemHash) {
+	  if (tempItemGradinglist == null) return false;
+	  if (tempItemGradinglist.isEmpty()) return false;
+	  
+	  Iterator iter = tempItemGradinglist.iterator();
+	  while(iter.hasNext()){
```

Comment:
Update this with a newer style loop (for, foreach, lambda)

Reply:
done!